### DISCUSSION TITLE
How to use my own database server?

### DISCUSSION THREAD

**Discussion by seventhsite** [photoprism/photoprism#1506] [Q&A] [answered]
Hi! I am already have a mysql-server on host and want to use it to save memory from docker mariadb. I created a DB and a user, but photoprism can't connect to it. Looks like there is needed some specific in docker-compose.yml? 
Now it looks like:

```
      # PHOTOPRISM_DATABASE_DRIVER: "sqlite"         # SQLite is an embedded database that doesn't require a server
      PHOTOPRISM_DATABASE_DRIVER: "mysql"            # Use MariaDB (or MySQL) instead of SQLite for improved performance
      PHOTOPRISM_DATABASE_SERVER: "localhost"      # MariaDB database server (hostname:port)
      PHOTOPRISM_DATABASE_NAME: "photoprism"         # MariaDB database schema name
      PHOTOPRISM_DATABASE_USER: "photoprism"         # MariaDB database user name
      PHOTOPRISM_DATABASE_PASSWORD: "mypassword"       # MariaDB database user password
```
And I commented all mariadb service. 
I also tried to put something like

```
PHOTOPRISM_DATABASE_SERVER: "localhost:3306"
PHOTOPRISM_DATABASE_SERVER: "my server private IP 192.168.7.2"
```

but get the same error in log:

`photoprism    | time="2021-08-31T13:42:52Z" level=fatal msg="dial tcp 192.168.7.2:3306: connect: connection refused"`

What I have to write in DB server option?

**Comment by bdronneau** [comment, marked as answer]
Hello,

If you run photoprism in Docker and your mysql is installed on your local system, you need to use something like this[this](https://stackoverflow.com/questions/44543842/how-to-connect-locally-hosted-mysql-database-with-the-docker-container/44544841) 

Beat
